lib/cisco_node_utils/cmd_ref/fabricpath.yaml

```diff
@@ -0,0 +1,178 @@
+# fabricpath
+---
+# Fabricpath feature is not available on N3K and N9K
+_exclude: [/N3/, /N9K/]
+
+aggregate_multicast_routes:
+  _exclude:
+    - /N5K/
+  kind: boolean
+  config_get: "show run fabricpath"
+  config_get_token: '/^fabricpath multicast aggregate\-routes$/'
+  config_set: "%s fabricpath multicast aggregate-routes"
```

Comment:
At the very least we need to be consistent, but in general we are migrating all existing code to use the key-value wildcards given that all new providers use that to align with our best practices for future extensibility.  I know it's a bit more work but I think it will pay off in the long run.